code/modules/power/cell.dm

```diff
@@ -277,7 +272,7 @@
 	icon_state = "icell"
 	maxcharge = 30000
 	materials = list(MAT_GLASS=1000)
-	rating = 6
+	rating = 100

```

Comment:
technically an unnanounced change?

Reply:
`fixes part ratings of cells so slime cells are correctly more desirable than bluespace cells and other such nonsense`